include/sys/arc_impl.h

```diff
@@ -899,7 +899,14 @@ typedef struct arc_stats {
 	kstat_named_t arcstat_abd_chunk_waste_size;
 } arc_stats_t;
 
+/*
+ * We place aggsum_t members first to reduce structure padding.
+ */
 typedef struct arc_sums {
+	aggsum_t arcstat_size;
+	aggsum_t arcstat_dnode_size;
+	aggsum_t arcstat_l2_hdr_size;
+	aggsum_t arcstat_meta_used;
```

Comment:
I worry that different order here will make it more difficult to maintain.  I am not sure it worth it.

Reply:
I am going to err on the side that it is not worth it given that the other change turned out to be a bad idea. I am closing this PR.